database/src/main/postgres/runs/measurements.ddl

```diff
@@ -13,20 +13,18 @@
  * limitations under the License.
  */
 
--- DROP TABLE IF EXISTS runs.measurements;
-
 CREATE TABLE runs.measurements
 (
-    id_measurement                      UUID NOT NULL,
-    key_checkpoint_measure_definition   BIGINT NOT NULL,
-    key_checkpoint                      UUID NOT NULL,
-    value                               JSON NOT NULL,
+    id_measurement                      BIGINT NOT NULL DEFAULT global_id(),
+    fk_measure_definition               BIGINT NOT NULL,
+    fk_checkpoint                       UUID NOT NULL,
+    measurement_value                   JSONB NOT NULL,
```

Comment:
I think here we could even create a composite type, because we already know what the structure should be. What do you think? More flexibility can be good in some cases, but can bring unexpected situations/bad values in others.

Not now, I just wanted to understand your reasoning